{"priority": "major", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "links": {"attachments": {"href": "data/repositories/osrf/gazebo/issues/2259/attachments_page=1.json"}, "self": {"href": "data/repositories/osrf/gazebo/issues/2259.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/2259/watch"}, "comments": {"href": "data/repositories/osrf/gazebo/issues/2259/comments_page=1.json"}, "html": {"href": "#!/osrf/gazebo/issues/2259/build-cop-integration_model_database"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues/2259/vote"}}, "reporter": {"display_name": "Shane Loretz", "uuid": "{656e3311-aad9-45a1-aaf7-b0ee0e84b287}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B656e3311-aad9-45a1-aaf7-b0ee0e84b287%7D"}, "html": {"href": "https://bitbucket.org/%7B656e3311-aad9-45a1-aaf7-b0ee0e84b287%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:684383ab-ac95-4859-a350-4a6f41a94a22/c7a1ebf5-cade-4115-9f26-9d3facb776db/128"}}, "nickname": "Shane Loretz", "type": "user", "account_id": "557058:684383ab-ac95-4859-a350-4a6f41a94a22"}, "title": "[build cop] INTEGRATION_model_database GetModelsTwice timeout", "component": null, "votes": 0, "watches": 1, "content": {"raw": "Creating an issue to track a possible flaky test. Please post a comment if you see it again. Probably a temporary networking issue so I'll rerun the build.\r\n\r\n[gazebo-ci-default-homebrew-amd64](https://build.osrfoundation.org/view/main/view/BuildCopTests/job/gazebo-ci-default-homebrew-amd64/) failed with a timeout after 400 seconds. Jenkins reports the problem as `INTEGRATION_model_database.test_ran` because it didn't get test results.\r\n\r\n```\r\n        Start 225: INTEGRATION_model_database\r\n\r\n225: Test command: /Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test/integration/INTEGRATION_model_database \"--gtest_output=xml:/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test_results/INTEGRATION_model_database.xml\"\r\n225: Environment variables: \r\n225:  CMAKE_PREFIX_PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build::/usr/local/opt/qt5\r\n225:  GAZEBO_PLUGIN_PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/plugins:/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/plugins/events:/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/plugins/rest_web\r\n225:  GAZEBO_RESOURCE_PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/gazebo9\r\n225:  PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/gazebo:/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/tools:/usr/local/opt/ccache/libexec:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/opt/X11/bin\r\n225:  PKG_CONFIG_PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/cmake/pkgconfig:\r\n225: Test timeout computed to be: 400\r\n225: [==========] Running 4 tests from 1 test case.\r\n225: [----------] Global test environment set-up.\r\n225: [----------] 4 tests from ModelDatabaseTest\r\n225: [ RUN      ] ModelDatabaseTest.GetModels\r\n225: [Msg] Waiting for master.\r\n225: [Msg] Connected to gazebo master @ http://127.0.0.1:11345\r\n225: [Msg] Publicized address: 172.23.2.252\r\n225: [Dbg] [ServerFixture.cc:209] ServerFixture load in 1 seconds, timeout after 600 seconds\r\n225: [Dbg] [ServerFixture.cc:135] ServerFixture::Unload\r\n225: [       OK ] ModelDatabaseTest.GetModels (157020 ms)\r\n225: [ RUN      ] ModelDatabaseTest.GetModelsTwice\r\n225/350 Test #225: INTEGRATION_model_database ............................***Timeout 400.07 sec\r\ntest 226\r\n        Start 226: check_INTEGRATION_model_database\r\n\r\n226: Test command: /Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/gazebo9/tools/check_test_ran.py \"/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test_results/INTEGRATION_model_database.xml\"\r\n226: Test timeout computed to be: 9.99988e+06\r\n226: Checking for test results in /Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test_results/INTEGRATION_model_database.xml\r\n226: Cannot find results, writing failure results to /Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test_results/INTEGRATION_model_database.xml\r\n226/350 Test #226: check_INTEGRATION_model_database ......................***Failed    0.03 sec\r\n```", "markup": "markdown", "html": "<p>Creating an issue to track a possible flaky test. Please post a comment if you see it again. Probably a temporary networking issue so I'll rerun the build.</p>\n<p><a data-is-external-link=\"true\" href=\"https://build.osrfoundation.org/view/main/view/BuildCopTests/job/gazebo-ci-default-homebrew-amd64/\" rel=\"nofollow\">gazebo-ci-default-homebrew-amd64</a> failed with a timeout after 400 seconds. Jenkins reports the problem as <code>INTEGRATION_model_database.test_ran</code> because it didn't get test results.</p>\n<div class=\"codehilite\"><pre><span></span>        Start 225: INTEGRATION_model_database\n\n225: Test command: /Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test/integration/INTEGRATION_model_database &quot;--gtest_output=xml:/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test_results/INTEGRATION_model_database.xml&quot;\n225: Environment variables: \n225:  CMAKE_PREFIX_PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build::/usr/local/opt/qt5\n225:  GAZEBO_PLUGIN_PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/plugins:/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/plugins/events:/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/plugins/rest_web\n225:  GAZEBO_RESOURCE_PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/gazebo9\n225:  PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/gazebo:/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/tools:/usr/local/opt/ccache/libexec:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/opt/X11/bin\n225:  PKG_CONFIG_PATH=/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/cmake/pkgconfig:\n225: Test timeout computed to be: 400\n225: [==========] Running 4 tests from 1 test case.\n225: [----------] Global test environment set-up.\n225: [----------] 4 tests from ModelDatabaseTest\n225: [ RUN      ] ModelDatabaseTest.GetModels\n225: [Msg] Waiting for master.\n225: [Msg] Connected to gazebo master @ http://127.0.0.1:11345\n225: [Msg] Publicized address: 172.23.2.252\n225: [Dbg] [ServerFixture.cc:209] ServerFixture load in 1 seconds, timeout after 600 seconds\n225: [Dbg] [ServerFixture.cc:135] ServerFixture::Unload\n225: [       OK ] ModelDatabaseTest.GetModels (157020 ms)\n225: [ RUN      ] ModelDatabaseTest.GetModelsTwice\n225/350 Test #225: INTEGRATION_model_database ............................***Timeout 400.07 sec\ntest 226\n        Start 226: check_INTEGRATION_model_database\n\n226: Test command: /Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/gazebo9/tools/check_test_ran.py &quot;/Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test_results/INTEGRATION_model_database.xml&quot;\n226: Test timeout computed to be: 9.99988e+06\n226: Checking for test results in /Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test_results/INTEGRATION_model_database.xml\n226: Cannot find results, writing failure results to /Users/jenkins/workspace/gazebo-ci-default-homebrew-amd64/build/test_results/INTEGRATION_model_database.xml\n226/350 Test #226: check_INTEGRATION_model_database ......................***Failed    0.03 sec\n</pre></div>", "type": "rendered"}, "assignee": null, "state": "new", "version": null, "edited_on": null, "created_on": "2017-04-27T20:24:24.446662+00:00", "milestone": null, "updated_on": "2017-04-27T20:43:10.330494+00:00", "type": "issue", "id": 2259}